### ISSUE TITLE
URL constructor in Deno 1.2

### ISSUE THREAD

**Issue by Bidek56** [sholladay/pogo#46]
In Deno 1.2, this simple code fails with **Type 'URL' is not assignable to type 'string'.**

```
import React from 'https://dev.jspm.io/react';
import pogo from 'https://deno.land/x/pogo/main.ts';

const server = pogo.server({ port : 3000 });

server.router.get('/', () => {
    return <h1>Hello, world!</h1>;
});

server.start();
```

Definition of URL constructor has changed in 1.2 by [PR](https://github.com/denoland/deno/pull/6653) and I think you need to upgrade your package. Thx

**Comment by afaur** [comment]
I looked into this, and created a PR (https://github.com/sholladay/pogo/pull/47) which seems to fix this.
More information available here: https://dev.to/srnv/deno-1-2-0-url-argument-type-solutions-37m0

**Comment by sholladay** [comment]
Fixed in PR #47
